account_invoice_line_sale_line_position/models/account_invoice.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    has_order_position = fields.Boolean(compute="_compute_has_order_position")
+
+    @api.depends("invoice_line_ids.position_formatted")
+    def _compute_has_order_position(self):
+        for record in self:
+            record.has_order_position = any(
+                record.invoice_line_ids.mapped("position_formatted")
+            )
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    position_formatted = fields.Char(compute="_compute_position_formatted")
+
+    @api.depends("sale_line_ids.position")
```

Comment:
I don't think this depends is required: this field will be computed when used anyway. Also, I'd avoid recomputing this every time the value on the SO lines change.

Reply:
I don't think the value on the sale.order.line will changed when the line is invoiced. As the sale order will have been sent and at that time the position will not change.
I can remove the `api.depends` if you think it is better